### DISCUSSION TITLE
Great app!

### DISCUSSION THREAD

**Discussion by linwaytin** [felixwiemuth/SimpleReminder#19] [General]
This app is really great! I like the idea that you can only schedule things for the next 24 hours. This "restriction" actually makes me focus on what I need to do now, instead of planing some other things for the future. 

I would suggest adding a button for "Mark done" when selecting a task, but this is not a big issue.

Thanks for this great app!

**Comment by felixwiemuth** [comment]
Thanks for your feedback!

I agree that focussing on the immediate future (e.g. next 24h) generally is a good idea; the app is no calendar, and misusing it for one can be counterproductive. One could, however, think of use cases where it can replace a calendar.

I still think one should have the possibility to add reminders a bit more ahead (and technically on arbitrary dates). It will be one's own responsibility to not overdo it - which in general can be a problem when adding too many reminders which maybe are not really necessary.
Regarding the design of this feature, you are welcome to comment on #10.

Regarding "Mark done": When you select one or multiple due or scheduled reminders, there is an action button "Mark done". It will usually appear as an entry in the menu because Android decides there is not enough space in the action bar. Did you mean anything else?

**Reply by linwaytin** [reply]
Oh so that's why I see it "Mark done" in the menu. Thanks for explaining.
